docs/build/abci++/00-introduction.md

```diff
@@ -0,0 +1,52 @@
+# Introduction
+
+## What is ABCI++?
+
+ABCI++, Application Blockchain Interface is the interface between CometBFT and the application. Within the release of ABCI 2.0 for the 0.38 CometBFT release there were additional methods introduced.
+
+The 5 methods introduced during ABCI 2.0 are:
+
+- `PrepareProposal`
+- `ProcessProposal`
+- `ExtendVote`
+- `VerifyVoteExtension`
+- `FinalizeBlock`
+
+
+## The Flow
+
+## PrepareProposal
+
+Based on their voting power, CometBFT chooses a block proposer and calls `PrepareProposal` on the block proposer's application (Cosmos SDK). The selected block proposer is responsible for collecting outstanding transactions from the mempool, adhering to the application's specifications. The application can enforce custom transaction ordering and incorporate additional transactions, potentially generated from vote extensions in the previous block.
+
+To perform this manipulation on the application side, a custom handler is implemented. If no custom handler exists, all transactions are validated. Please see [this](https://github.com/fatal-fruit/abci-workshop) tutorial for more information on custom handlers.
```

Comment:
```suggestion
To perform this manipulation on the application side, a custom handler must implemented. By default, the Cosmos SDK provides a `PrepareProposalHandler`, which if used with a application specific mempool. A custom handler can be written by application developer, if a noop handler provided, all transactions are considered valid. Please see [this](https://github.com/fatal-fruit/abci-workshop) tutorial for more information on custom handlers.
```

Reply:
> To perform this manipulation on the application side, a custom handler must be implemented. By default, the Cosmos SDK provides a `PrepareProposalHandler`, >>which if used with a application specific mempool<<. A custom handler can be written by the application developer, but if a noop handler is provided, all transactions are considered valid


I added some connectors that I think were missing, I still don't know what you meant with "which if used with a application specific mempool"